sda/internal/database/db_functions.go

```diff
@@ -857,3 +858,28 @@ func (dbs *SDAdb) DeprecateKeyHash(keyHash string) error {
 
 	return nil
 }
+
+func (dbs *SDAdb) GetReVerificationData(accessionID string) (schema.IngestionVerification, error) {
+	dbs.checkAndReconnectIfNeeded()
+	db := dbs.DB
+	reVerify := schema.IngestionVerification{ReVerify: true}
+
+	const query = "SELECT archive_file_path,id,submission_file_path,submission_user FROM sda.files where stable_id = $1;"
+	err := db.QueryRow(query, accessionID).Scan(&reVerify.ArchivePath, &reVerify.FileID, &reVerify.FilePath, &reVerify.User)
+	if err != nil {
+		return schema.IngestionVerification{}, err
```

Comment:
Would it be better to include `accessionID` in the error message for easier debugging?